style.css

```diff
@@ -324,7 +328,29 @@ ul {
   border-radius: 12px;
 }
 
+#portfolio {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #f1f2f5;
+  padding: 80px 0;
+  gap: 70px;
+}
+
 @media screen and (min-width: 768px) {
+  .port1 {
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .title {
+    font-weight: 700;
+    font-size: 32px;
+    color: #172b4d;
+    margin-bottom: 0;
+    margin-top: 10px;
+  }
+
```

Comment:
[Optional] You have rightly implemented the error display. Multiple clicks of the submit button displays the error message multiple times. I suggest that you make it display once.

Also, by convention, error messages are displayed with the red colour of text to signify a warning. I recommend that you use it to also make it visible.

![image](https://user-images.githubusercontent.com/20554583/216360185-e29e7e53-4c2c-4ef6-8a46-3fa9a298324d.png)